Configs/.zshrc

```diff
@@ -73,5 +73,7 @@ alias ssh='kitten ssh'
 # To customize prompt, run `p10k configure` or edit ~/.p10k.zsh.
 [[ ! -f ~/.p10k.zsh ]] || source ~/.p10k.zsh
 
-#Display Pokemon
-pokemon-colorscripts --no-title -r 1,3,6
+# Pokemon sprites, aren't they cute?
+if [[ $TERM == 'xterm-kitty' ]]; then
```

Comment:
thanks, but what if the user has a different terminal.
I don't think there is a need to hardcode the config to work for a specific term, so users can disable it manually if not required.